src/rabbit_auth_backend_http.erl

```diff
@@ -163,6 +166,9 @@ escape(K, V) ->
 
 parse_resp(Resp) -> string:to_lower(string:strip(Resp)).
 
+join_tags([]) -> "";
+join_tags(Tags) -> string:substr(lists:foldr(fun(Elem, Acc) ->  ", " ++ atom_to_list(Elem) ++ Acc end, "", Tags), 3).
```

Comment:
I'd say give this a try - http://erlang.org/doc/man/string.html#join-2

Reply:
I am finishing this PR.